tests/e2e/framework/kubernetes.go

```diff
@@ -154,13 +156,23 @@ func newKubeInfo(tmpDir, runID, baseVersion string) (*KubeInfo, error) {
 	} else {
 		releaseDir = util.GetResourcePath("")
 	}
+	// Note the kubectl commands used by the test will default to use the local
+	// environments kubeconfig if an empty string is provided.  Therefore in the
+	// default case kubeConfig will not be set.
 	var kubeConfig, remoteKubeConfig string
 	var kubeClient, remoteKubeClient kubernetes.Interface
 	var aRemote *AppManager
 	if *multiClusterDir != "" {
-		// ClusterRegistiresDir indicates the Kubernetes cluster config should come from files versus KUBECONFIG
-		// environmental variable.  The test config can be defined to use either a single cluster or 2 clusters
-		kubeConfig, remoteKubeConfig, err = util.GetKubeConfigFromFile(*multiClusterDir)
+		// multiClusterDir indicates the Kubernetes cluster config should come from files versus
+		// the environmental. The test config can be defined to use either a single cluster or
+		// 2 clusters
+		tmpfile := *namespace + "_kubeconfig"
+		tmpfile = path.Join(tmpDir, tmpfile)
+		if err = util.GetKubeConfig(tmpfile); err != nil {
```

Comment:
Nit. A bit confused with the name GetKubeConfig(). I realized that it's actually creating a kubeconfig file from the in-cluster kube environment.